src/apis/AllowlistIdentifierApi.ts

```diff
@@ -0,0 +1,39 @@
+import { AbstractApi } from './AbstractApi';
+import { AllowlistIdentifier } from '../resources/AllowlistIdentifier';
+
+const basePath = '/allowlist_identifiers';
+
+type CreateParams = {
```

Comment:
🙃 
I would opt for naming the params something like:
`AllolistIdentifierCreateParams` just for the searchability, WDYT ?

Reply:
:+1: Done.